### ISSUE TITLE
When "resetting" a client, reset a $facets array too

### ISSUE THREAD

**Issue by fruit** [FoolCode/SphinxQL-Query-Builder#110]
When I run multiply queries one after one, I'm getting the previous facets results returned.

```
$conn = $client->getConnection();
$next = $client
    ->select(['id', 'title])
    ->from('my_index')
    ->orderBy('my_count', 'DESC')
    ->facet(Facet::create($conn)->facet(['cat_a'])->orderBy('COUNT(*)', 'DESC'))
    ->enqueue(SphinxQL::create($conn)->query('SHOW META'));

$next->executeBatch()->getStored();  // returns SELECT + FACET + META (got 3 results, expected 3 results)

$client->reset()->select(['id'])->from('my_index')->limit(1);

$client->executeBatch()->getStored(); // returns SELECT + FACET (got 2 results, expected 1 result)
```

I would like to have the resetting of $facets within a `reset()` method call.

**Comment by woxxy** [comment]
Requires adding a line to https://github.com/FoolCode/SphinxQL-Query-Builder/blob/master/src/SphinxQL.php#L1346 and possibly a test to avoid regression.

**Comment by oohnoitz** [comment]
implemented in 7e2ade807ee78d4667d0bf1b4200288b347601ef
